pkgs/tools/misc/rauc/default.nix

```diff
@@ -22,21 +27,29 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-VpHcJUTRZ5aJyfYypjVsYyRNrK0+9ci42mmlZQSkWAk=";
   };
 
+  patches = [
+    (fetchpatch {
+      # Patch to install the man page when using meson, remove on package bump
+      url = "https://github.com/rauc/rauc/commit/756c677d031c435070a6900e6778d06961822261.patch";
+      hash = "sha256-QgIUagioRo61PeC0JyKjZtnauFiYP1Fz9wrxGEikBGI=";
+    })
+  ];
   passthru = {
     updateScript = nix-update-script { };
   };
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  nativeBuildInputs = [ pkg-config meson ninja ];
 
-  buildInputs = [ curl dbus glib json-glib openssl ];
+  buildInputs = [ curl dbus glib json-glib openssl utillinux libnl systemd ];
```

Comment:
util-linux usually goes to nativeBuildInputs

Reply:
`rauc` requires fdisk as a run-time dependency, so in this case `buildInputs` should be correct.